src/webserver/details/HttpBase.h

```diff
@@ -207,21 +207,10 @@ class HttpBase : public ConnectionBase
 
         log_.info() << tag() << "Received request from ip = " << clientIp << " - posting to WorkQueue";
 
-        auto request = boost::json::object{};
         try
         {
-            request = boost::json::parse(req_.body()).as_object();
-        }
-        catch (boost::exception const& e)
-        {
-            return sender_(httpResponse(
-                http::status::ok,
-                "application/json",
-                boost::json::serialize(RPC::makeError(RPC::RippledError::rpcBAD_SYNTAX))));
-        }
-        try
-        {
-            (*handler_)(std::move(request), derived().shared_from_this());
+            std::string reqStr = req_.body();
+            (*handler_)(std::move(reqStr), derived().shared_from_this());
```

Comment:
Looks like we are attempting to move the request string here but the receiving end is now `const&`. I think we can actually do the move as `reqStr` is not used anymore in this scope. Consider updating either here or in the executor 👍

Reply:
Good catch . the last minute i changed the && to const&. missed this change, oops